charts/mimir-distributed/CHANGELOG.md

```diff
@@ -0,0 +1,239 @@
+# Changelog
+
+This changelog is continued from `enterprise-metrics` after Grafana Enterprise Metrics was added to `mimir-distributed` in PR #1203.
+All notable changes to this chart will be documented in this file.
+
+Entries should be ordered as follows:
+- [CHANGE]
+- [FEATURE]
+- [ENHANCEMENT]
+- [BUGFIX]
+
+Entries should include a reference to the Pull Request that introduced the change.
+
+## 2.0.0
+
+* [CHANGE] **Breaking** for existing users of `mimir-distributed`: the naming convention is changed to have shorter resource names, as in `<release>-mimir-distributed-store-gateway` is now just `<release>-mimir-store-gateway`. To have the previous names, please specify `nameOverride: mimir-distributed` in the values. #1203
+* [CHANGE] The chart `enterprise-metrics` is renamed to `mimir-distributed`. #1203
+* [CHANGE] **Breaking** Configuration for Grafana Enterprise Metrics is now in the value `mimir.config` as a helm template **string**.
+  Please consult the [Grafana Enterprise Migration Guide](https://grafana.com/docs/enterprise-metrics/latest/) to learn more about how to upgrade the configuration.
+  Except for the following parameters specified as command line parameters in the Pod templates,
+  everything is now set in this string-typed value, giving a definitive source of configuration.
+  Exceptions:
+    > The `-target=` must be provided individually.\
+    The `-config.file=` obviously.\
+    The `-activity-tracker` for convenience.\
```

Comment:
I'm curious why this one.

Reply:
I wanted to have a single configuration for this and keep it in the config, but I got a comment that I should set it for services that don't use it (i.e. anything on write path).

Reply:
It turns out to be incompatible with the mimirtool convert, so this is now removed and fixed.